frontend/src/core/user/components/UserMenu.js

```diff
@@ -0,0 +1,208 @@
+/* @flow */
+
+import * as React from 'react';
+import onClickOutside from 'react-onclickoutside';
+import { Localized } from 'fluent-react';
+
+import './UserMenu.css';
+
+import SignOut from './SignOut';
+
+import type { NavigationParams } from 'core/navigation';
+import type { UserState } from 'core/user';
```

Comment:
In imports, I usually put local imports after module imports, so those should be one block up. I'm not sure I have written those rules down anywhere, if I haven't, I should. Feel free to remind me to do so. :stuck_out_tongue:

Reply:
Oh, that's OK. For some reason I thought `type` imports are an exception and always come in last.

Reply:
Damn, sorry, I didn't see those were type imports... You were absolutely correct. :-(

Reply:
NP, I'll fix it in https://bugzilla.mozilla.org/show_bug.cgi?id=1534708.